### PR TITLE
Create my project referral yaml file!

### PR DESCRIPTION
**Pull request by orbitalsupershell**

### CODE REVIEW

File: manifests/sensor.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: sensor
+authors: ORBIT
+homepage: https://github.com/orbitalsupershell/sensor/
+license: ALL RIGHTS RESERVED
```
Comment (samrocketman) [inline manifests/sensor.yaml:4]: I notice your plugin copies some assets from the base game and there's no debian copyright file with attribution.

Please add a copyright file with attribution and sources for the images you copied.

**Review by samrocketman — CHANGES_REQUESTED** [review]

File: manifests/sensor.yaml (hunk 1 above)
Comment (orbitalsupershell) [reply manifests/sensor.yaml:4]: Working on a newer plugin, consider this PR closed.

**Comment by orbitalsupershell** [comment]
Please review my project, big changes have been done.

File: manifests/sensor.yaml (hunk 1 above)
Comment (orbitalsupershell) [reply manifests/sensor.yaml:4]: Please review copyright @samrocketman

Comment (samrocketman) [reply manifests/sensor.yaml:4]: Looks ok to me.

**Review by samrocketman — APPROVED** [review]
License seems fine since vanilla material removed from plugin.

In your plugin:

* Rename `About.txt` to lowercase `about.txt` or it will break for other platforms such as Linux.

**Comment by orbitalsupershell** [comment]
Fixed, thanks for pointing it out.

**Pull request merged**